site/docs/components/stepper-input/accessibility.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Best practices
+
+- Wrap `StepperInput` in [`FormField`](/salt/components/form-field) to provide the field with a visible label, help text and a status message for validation feedback. This will ensure it complies with accessibility guidelines. You can find more information [in the W3C form instructions](https://www.w3.org/WAI/tutorials/forms/instructions/).
+- `aria-valuenow`, `aria-valuemax`, and `aria-valuemin` attributes are set to provide users with information about the current value, and the allowable range of values.
```

Comment:
Is this a best practice? Maybe something to look at in the future. This seems more like a spec of how the component works.

Reply:
Removed